test/ts-use/package.json

```diff
@@ -2,9 +2,18 @@
   "name": "ts-use",
   "private": true,
   "dependencies": {
-    "multiformats": "file:../../dist/"
+    "multiformats": "file:../../"
   },
   "scripts": {
     "test": "npm install && npm_config_yes=true npx -p typescript tsc --noEmit"
```

Comment:
IF this is using npx, should we pin to a specific version of typescript?

Might cause issues down the line.

Reply:
yeah .. except then it needs to stay up to date, so it'd need to become a dependency in here and then have the dependabot config check this file

_or_ we could just reach back into `../node_modules/.bin/tsc` since we know it's there and the version will stay pinned to whatever the parent is using